JASP-Desktop/html/index.html

```diff
@@ -27,7 +27,7 @@
 	<div id="templates" ></div>
 	<div id="intro">
 		<h1>Welcome to JASP!</h1>
-		<p style="text-align: right ;">Version 0.7 Beta 4</p>
+		<p style="text-align: right ;">Version <span class="app_version">Unknown</span></p>
```

Comment:
i'm pretty sure we don't use underscores in class names, or do we?
and perhaps this could be unpopulated/blank until setAppVersion() is called.

i'm invisaging people taking and using our result renderer in other projects in the future, and it would be a bit easier if they could ignore setAppVersion() if it wasn't important to them.